.github/workflows/phpunit.yml

```diff
@@ -30,14 +30,13 @@ jobs:
       matrix:
         # quotes are needed it is treated as a number and zero at decimal part is gone at runtime
         laravel: ["9.x"]
-        # PHP 8.0 should be covered as well, but I switched to PHPUnit 10 and that requires PHP 8.1
-        php: ["8.1"]
+        php: ["8,0", "8.1"]
```

Comment:
@MarcinOrlowski There is a comma here 👆🏻

Reply:
Damn. Thanks! 👏🏻 Things like this were the reason I had my previous IDE configured so it highlighted dots differently than commas.